posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -141,6 +183,18 @@ def event_type_expr(self) -> ast.Expr:
 
         if self.conversion_goal_action:
             return ast.Call(name="or", args=[pageview_expr, action_to_expr(self.conversion_goal_action)])
+        elif self.conversion_goal_custom_event:
+            return ast.Call(
+                name="or",
+                args=[
+                    pageview_expr,
+                    ast.CompareOperation(
+                        left=ast.Field(chain=["event"]),
+                        op=ast.CompareOperationOp.Eq,
+                        right=ast.Constant(value=self.conversion_goal_custom_event),
+                    ),
```

Comment:
Guessing we can't use `entity_to_expr` here?

Reply:
Maybe we can abstract it to a function in this file

Reply:
Hmm I wasn't aware of that. It's using a `RetentionEntity`, it looks like the code itself does what I would want, but the data model has some extra stuff in it that I wouldn't.

I think your suggestion of creating a function in this file would work better